app/views/storytime/dashboard/posts/_list.html.erb

```diff
@@ -3,7 +3,12 @@
     <%= link_to [:edit, :dashboard, post], id: "#{post.class.to_s.downcase.split("::").last}_#{post.id}", class: "list-group-item" do %>
       <div class="row">
         <div class="col-md-8">
-           <h4><%= post.title %></h4>
+          <h4><%= post.title %></h4>
+        </div>
+        <div class="col-md-2 col-md-offset-2">
+          <%= form_tag(url_for([:dashboard, post]), method: :delete) do |f| %>
```

Comment:
the markup would be something like (in HAML since it's easier to type here):

```
%ul.list-group.list-group-button-links
    %li.list-group-item
        = link_to [:edit, :dashboard, post], id: [same as before], class: "list-group-item" do
           = post.title
        %div.list-group-actions
            [delete link goes here]
```